src/Aspire.AppHost.Sdk/SDK/Sdk.in.targets

```diff
@@ -91,52 +93,19 @@
     </PropertyGroup>
 
     <!-- At this point, we should have the version either by CPM or PackageReference, so we fail if not. -->
-    <Error Condition="'$(_AppHostVersion)' == ''" 
+    <Error Condition="'$(_AppHostVersion)' == ''"
        Text="$(MSBuildProjectName) is a .NET Aspire AppHost project that needs a package reference to Aspire.Hosting.AppHost version 8.2.0 or above to work correctly." />
 
     <!--The following is the list of the RIDs that we currently multitarget for DCP and Dashboard packages-->
     <ItemGroup>
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x86" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-arm64" />
+      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64;win-x86;win-arm64;linux-x64;linux-arm64;osx-x64;osx-arm64" />
```

Comment:
this is a nitpick kind of thing - when the items are declared inline like this they show as one 'thing' in the binlog viewer:
![image](https://github.com/user-attachments/assets/69ea1ae1-8cae-4017-a818-0702c38ed535)

this is easier to look at than N separate items.